lib/can_do.rb

```diff
@@ -26,8 +28,8 @@
 #
 class CanDo
   include Singleton
-
-  NotAFeature = Class.new(StandardError)
+  FEATURE_KEY_PREFIX = "features/"
+  CONNECTION_POOL = ConnectionPool.new(size: 5, timeout: 5) { Redis.new(url: ENV.fetch('CANDO_REDIS_URL', nil)) }
```

Comment:
how does this behave if the env var is empty? does `Redis.new(url: nil)` crash? (we probably wouldn't want that)

Reply:
should we detect from beginning and fallback?